.changes/next-release/Feature-62e021c0-3868-4f50-b438-b7c1a47086c2.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "User can select an S3 bucket when deploying a SAM app"
```

Comment:
Consider avoiding references to the user as a third party in changelogs. For example:

"When deploying a SAM application, the S3 bucket is now chosen from a list. Previously, the bucket name had to be typed in."